language/use-cases/document-qa/question_answering_documents_langchain_bigquery_vector_search.ipynb

```diff
@@ -0,0 +1,1026 @@
+{
```

Comment:
<p>Formatting add space after <code>#</code></p><p> </p><p><code>And add link to Colab Enterprise, like how this notebook is set up </code><a href="https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb" target="_blank">https://github.com/GoogleCloudPlatform/generative-ai/blob/main/gemini/getting-started/intro_gemini_python.ipynb</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/494/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done.</p>